src/current/v24.1/backup.md

```diff
@@ -97,6 +97,9 @@ Query parameter | Value | Description
 ----------------+-------+------------
 `ASSUME_ROLE` | [`STRING`]({% link {{ page.version.version }}/string.md %}) |{% include {{ page.version.version }}/misc/assume-role-description.md %} Refer to [Cloud Storage Authentication]({% link {{ page.version.version }}/cloud-storage-authentication.md %}) for setup details.
 `AUTH` | [`STRING`]({% link {{ page.version.version }}/string.md %}) | The authentication parameter can define either `specified` (default) or `implicit` authentication. To use `specified` authentication, pass your [Service Account](https://cloud.google.com/iam/docs/understanding-service-accounts) credentials with the URI. To use `implicit` authentication, configure these credentials via an environment variable. Refer to the [Cloud Storage Authentication page]({% link {{ page.version.version }}/cloud-storage-authentication.md %}) page for examples of each of these.
+`AWS_ENDPOINT` | Specify a custom endpoint for Amazon S3 or S3-compatible services. Use to define a particular region or a Virtual Private Cloud (VPC) endpoint.
+`AWS_SESSION_TOKEN` | (Optional) Use as part of temoporary security credentials when accessing AWS S3. For more information, refer to Amazon's guide on [temporary credentials](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_temp_use-resources.html).
+{% include_cached new-in.html version="v24.1.10" %} `AWS_USE_PATH_STYLE` | Change the URL format to path style from the default AWS S3 virtual-hosted–style URLs when connecting to Amazon S3 or S3-compatible services.
```

Comment:
So, not exactly - that's true on 24.3+. But on 24.1/24.2, the param exists but does nothing. Path style is actually the _only_ available system on those versions. For 24.3, we changed the behavior. We added `AWS_USE_PATH_STYLE` to force the old behavior on 24.3+, and backported the param as a noop to 24.1/24.2 so users who need it could set it in advance of upgrading.